contracts/extensions/collections/HasProxyAdmin.sol

```diff
@@ -4,18 +4,32 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/utils/StorageSlot.sol";
 
 abstract contract HasProxyAdmin {
+  error Unauthorized();
```

Comment:
Should rename the event for the the ease of debugging, and add `Err-` prefix. Maybe `ErrNotProxyAdmin`.

Reply:
resolved.